src/TraceEvent/Parsers/KernelTraceEventParser.cs

```diff
@@ -7597,7 +7597,7 @@ protected internal override Delegate Target
         }
         protected internal override void Dispatch()
         {
-            Debug.Assert(!(Version >= 1 && EventDataLength < HostOffset(52, 1)));
+            // Debug.Assert(!(Version >= 1 && EventDataLength < HostOffset(52, 1)));
```

Comment:
Once you're ready to merge, please undo the changes to the asserts.  I am trying to work on this separately.